LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -255,14 +271,27 @@ void loop() {
     }
 
     secsTillTx -= round((millis()-loop_start)/1000);
+    secsTillPing -= round((millis()-loop_start)/1000);
+    #if defined(DEVMODE)
+      Serial.println(round((millis()-loop_start)/1000));
+    #endif
   } else {
     secsToCheckBatt--;
 
     secsToCheckBatt -= (millis()-loop_start)/1000;
     // sleepSeconds(BattWait-((millis-loop_start)/1000));
   }
-  sleepSeconds(secsTillTx);
-  secsTillTx = 0;
+  // Serial.println("Loop time in milliseconds->");
+  // Serial.println(round((millis()-loop_start)/1000));
+  int sleepSecs;
```

Comment:
let's move this to the start of the function and not define it in the middle.

Reply:
Why would we need to move it to the start? It doesn't get used till the very end, so why define it up at the beginning? Wouldn't that make the code more confusing?

Reply:
in C, many compilers would not even allow an inline declaration.  it is typically at the start of a brace segment, or beginning of the function. 

if (x==5) {
   int myVar;
   <blah>
}

but doing:

<blah>
int myVar;

<blah>

is generally not permitted/suggested/recommended and is, stylewise, not the best.